ui/desktop/app/routes/scopes.js

```diff
@@ -23,9 +44,16 @@ export default class ScopesRoute extends Route {
    * @param {string} params.scope_id
    * @return {Promise{[ScopeModel]}}
    */
-  model() {
+  async model() {
     // NOTE:  In the absence of a `scope_id` query parameter, this endpoint is
     // expected to default to the global scope, thus returning org scopes.
-    return this.store.query('scope', {}).catch(() => A([]));
+    let scopes = A([]);
+    const isClientDaemonSupported = get(this, 'isClientDaemonSupported');
+
+    if (isClientDaemonSupported) {
```

Comment:
I don't understand why this check is needed? Don't we need to retrieve the scopes regardless?